utils/subfiling_vfd/h5fuse.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Isn't this the post configuration version of the .in file?

Reply:
Ah yes. Its checkin might have predated creation of the .in file. Will remove